src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
@betanalpha, I got confirmation from @jgabry. It'll break existing things, but not hard to change.

What's `E__`? Could we rename that to something descriptive?